eng/build.sh

```diff
@@ -46,6 +45,7 @@ usage()
   echo "  --framework                Build framework: netcoreapp5.0 or net472 (short: -f)"
   echo "  --coverage                 Collect code coverage when testing"
   echo "  --testscope                Test scope, allowed values: innerloop, outerloop, all"
+  echo "  --testnobuild              Skip building tests when invoking -test"
```

Comment:
I don't see --testnobuild switch being parsed and honored in the args parsing.

Reply:
Is that an arcade thing?

Reply:
It is handled here: https://github.com/dotnet/runtime/blob/master/eng/build.ps1#L155. This gets converted to `/p:TestNoBuild`. This is the backing property of the dotnet cli command `dotnet test --no-build` which we will use with vstest as soon as we switch over to it. I kept the name as close as possible.

Reply:
Yeah but this is the `.sh` script, I might be missing something obvious, but I don't see that automatic handling to convert to MSBuild properties in the `.sh` script.

Reply:
oh right, thanks :)